server/routes/api.js

```diff
@@ -12,12 +12,12 @@
 
 'use strict';
 
+require('dotenv').config();
```

Comment:
I believe you only need to call this once at the start of the app, as it will turn all variables in the `.env` file into environment variables. These are then available for the entire process.

Citing the [docs](https://www.npmjs.com/package/dotenv#usage):

> As early as possible in your application, require and configure dotenv.